ext/dom/document.c

```diff
@@ -2141,7 +2138,7 @@ PHP_METHOD(DOMDocument, registerNodeClass)
 	}
 
 	if (ce == NULL || instanceof_function(ce, basece)) {
-		DOM_GET_OBJ(docp, id, xmlDocPtr, intern);
+		DOM_GET_INTERN(id, intern);
```

Comment:
Is it me, or is ``id`` always just ``ZEND_THIS`` ?

Reply:
Most of the time yes. I'll fold that in.